Formula/a/anyzig.rb

```diff
@@ -0,0 +1,34 @@
+class Anyzig < Formula
+  desc "Universal zig executable that runs any version of zig"
+  homepage "https://github.com/marler8997/anyzig"
+  url "https://github.com/marler8997/anyzig/archive/refs/tags/v2025_08_13.tar.gz"
+  version "2025_08_13"
+  sha256 "13511963d0dc570f5fe47ec83a23ff2982b53f1516076014aab6ec54638c0f3e"
+  license "MIT"
+
+  no_autobump! because: :incompatible_version_format
```

Comment:
```suggestion
```
`brew livecheck` seems to work fine, now that we've set the `version` to be consistent with what you're doing upstream:
```
❯ brew livecheck --debug anyzig

Formula:          anyzig
livecheck block?: No

URL:              https://github.com/marler8997/anyzig/archive/refs/tags/v2025_08_13.tar.gz
URL (processed):  https://github.com/marler8997/anyzig.git
Strategy:         Git

Matched Versions:
2025_02_10, 2025_02_21, 2025_03_05, 2025_03_06, 2025_04_24, 2025_05_24, 2025_05_25, 2025_06_07, 2025_08_03, 2025_08_13

anyzig: 2025_08_13 ==> 2025_08_13
```